monero-outreach-docs/translations/es/storyboard/refuge_es.md

```diff
@@ -0,0 +1,94 @@
+
+Partiendo del sigo XVI y con raíces en las leyes marítimas británicas, el decomiso civil se ha vuelto una industria billonaria para el gobierno durante los últimos 30 años. Con un apoyo estatal cada vez mayor, pasaron de una incautación aproximada de $93.7 millones de USD en activos y dinero, a un valor de más de [$2.5 billones de USD para el 2010](https://www.wsj.com/news/articles/SB10001424053111903480904576512253265073870). 
+
+Y mientras pueda que exista mérito, causa probable o preponderancia de la evidencia a favor de la incautación de algunos activos (dentro de EEUU los estados difieren frente a las normas sobre las pruebas de las leyes de decomiso), los cálculos a través de los años, [estiman que un 85%](http://www.sfgate.com/news/nation-world/article/Asset-forfeiture-both-an-effective-tool-4546043.php) de las instancias de decomiso civil realizadas son injustificables y superficiales. En los EEUU, tan solo en el 2010, se estima que hubo un promedio de 11,000 decomisos de casos no criminales, de los cuales solamente una décima parte de ellos fueron alegados en cortes federales. Un cálculo realizado por un escritor de SFgate, Stewart M. Powell, indica que el retorno de los decomisos federales a sus dueños se [aproximan a un escaso 1%](https://www.sfgate.com/news/nation-world/article/Asset-forfeiture-both-an-effective-tool-4546043.php). 
+ 
+Cabe anotar que en ciertos estados de los EEUU existe el llamado ["Equitable Sharing" o intercambio equitativo](https://www.justice.gov/criminal-mlars/equitable-sharing-program), un programa el cual procede a distribuir entre el estado y las autoridades federales los activos liquidados a través de la incautación y el decomiso. En el 2010, más de [$500 millones de USD fueron compartidos entre oficiales y agencias](https://www.wsj.com/articles/SB10001424053111903596904576514583716653122) a través del programa de intercambio equitativo, con de más de $5 billones de USD distribuidos desde la inauguración del programa en 1984; un "[slush fund](https://www.merriam-webster.com/dictionary/slush%20fund)" (fondo o dádiva para sobornar al "investido", político). Si estás creyendo que estos fondos son ejercidos solamente para los "avances" de las actividades políticas, puedes estar muy equivocado. Los funcionarios y oficiales encargados de las políticas públicas han utilizado estos fondos de "intercambio equitativo" para [subsidiar comidas, fiestas costosas y gastos personales](https://harvardlawreview.org/2018/06/how-crime-pays-the-unconstitutionality-of-modern-civil-asset-forfeiture-as-a-tool-of-criminal-law-enforcement/). 
+ 
+Para la persona promedio sometidas a los bloques legales de los derechos civiles, el recurso de apelación es difícil de conseguir. Y, el incentivo para excederse listo para aquellos dentro de esos ámbitos legales. 

```

Comment:
You may use _**la persona promedio sometida**_, or _**las personas promedio sometidas**_.

+ Y, el incentivo para excederse _**está**_ listo para aquellos dentro de esos ámbitos legales.

Reply:
Yes I missed plural there, thanks.